ooniprobe/AppDelegate.mm

```diff
@@ -79,54 +78,17 @@ -(void)handleNotification:(NSDictionary*)userInfo :(UIApplication *)application{
     UIApplicationState state = [application applicationState];
     if (state == UIApplicationStateActive)
     {
-        if ([type isEqualToString:@"open_href"]){
```

Comment:
can we remove the whole method? what are we doing in it now?

Reply:
The whole method is needed to handle notification